andela/balder-ah-backend
# #162979594 Fix deprecation warning errors with Social Auth implementation

#### What does this PR do?

Fixes some deprecation warnings in the console having to do with express sessions

#### Description of Task to be completed?

- update express session config to fix deprecation warnings shown in the console upon starting up the app.

#### How should this be manually tested?

- Clone the repo to your local machine
- Rename the `.env.sample` file to `.env` and fill out necessary details
- Run `npm run dev:start`
- Upon successful startup, you shouldn't see any deprecation warnings from `express-sessions`

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories? (If applicable)

#162979594

#### Screenshots (If applicable)

![image](https://user-images.githubusercontent.com/15332525/50711472-9beda300-106e-11e9-8d0e-0f8541e9db8d.png)


#### Questions:

N/A
